gamemode/modules/base/cl_fonts.lua

```diff
@@ -118,5 +118,3 @@ local function loadFonts()
         font = "ChatFont"})
 end
 loadFonts()
--- Load twice because apparently once is not enough
-hook.Add("InitPostEntity", "DarkRP_LoadFonts", loadFonts)
```

Comment:
Not sure how this "errors". Do test the keypad checker. It was the reason this was added.

Reply:
It does error on dev branch